newIDE/app/src/MainFrame/index.js

```diff
@@ -327,13 +351,26 @@ class MainFrame extends React.Component<Props, State> {
         );
       },
       err => {
-        showErrorBox(
-          i18n._(
-            t`Unable to open this project. Check that the path/URL is correct, that you selected a file that is a game file created with GDevelop and that is was not removed.`
-          ),
-          err
-        );
-        return;
+        if (fs.existsSync(autoSavePath)) {
```

Comment:
As discussed in other comment, let's use here too a `LocalProjectOpener.checkAutosave` or `LocalProjectOpener.hasAutosave` function